File: Generated_Codes_Check/Results/python/55.py

*/

import java.util.Scanner;
class Palindrome {
    public static void main(String args[])
    {
      Scanner in = new Scanner(System.in);
	  int test = in.nextInt();
	  while(test-->0){
		  int num = in.nextInt();
		  int temp = num;
		  int sum=0;
		  while(temp > 0){
			  int rem = temp % 10;
			  sum = sum*10 + rem;
			  temp = temp/10;
		  }
		  if(sum==num){
			  System.out.println("wins");
		  }
		  else{
			  System.out.println("losses");
		  }
	  }
	}
}

File: Scripts/generated_codes/java/55.java
/*
Explanation

In the first case, 331 is not a palindrome as reverse of 331 is 133.
In the second case, 666 is a palindrome and so the player wins.

*/
import java.util.*;
public class Hello {

	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);
		int t = sc.nextInt();
		for(int i=0;i<t;i++){
		    String n = String.valueOf(sc.nextInt());
		    String rev = "";
		    for(int j=n.length()-1;j>=0;j--){
		        rev += n.charAt(j);
		    }
		    System.out.println(n.equals(rev)?"wins":"losses");
		}
		sc.close();
	}
}